numpyro/distributions/continuous.py

```diff
@@ -1188,6 +1188,70 @@ def infer_shapes(
         return batch_shape, event_shape
 
 
+class CAR(Distribution):
+    arg_constraints = {
+        "loc": constraints.real_vector,
+        "alpha": constraints.unit_interval,
+        "tau": constraints.positive,
+        "W": constraints.positive_definite,
+    }
+    support = constraints.real_vector
+    reparametrized_params = [
+        "loc",
+        "alpha",
+        "tau",
+    ]
+
+    def __init__(
+        self,
+        loc=0.0,
+        alpha=None,
+        tau=None,
+        W=None,
+        validate_args=None,
+    ):
+        if jnp.ndim(loc) == 0:
+            (loc,) = promote_shapes(loc, shape=(1,))
+        # temporary append a new axis to loc
+        loc = loc[..., jnp.newaxis]
+        if W is not None:
+            loc, self.W = promote_shapes(loc, W)
+
+        batch_shape = lax.broadcast_shapes(jnp.shape(loc)[:-2], jnp.shape(self.W)[:-2])
+        event_shape = jnp.shape(self.W)[-1:]
+        self.loc = loc[..., 0]
+        self.alpha = alpha
+        self.tau = tau
+        super(CAR, self).__init__(
+            batch_shape=batch_shape,
+            event_shape=event_shape,
+            validate_args=validate_args,
+        )
+
+    def sample(self, key, sample_shape=()):
+        # MVN with covariance matrix inv(tau * (D - alpha * W))
+        pass
+
+    @validate_sample
+    def log_prob(self, value):
+        D = self.W.sum(axis=-1)
+
+        D_rsqrt = D ** (-0.5)
+        W_scaled = self.W * (D_rsqrt * D_rsqrt[:, jnp.newaxis])
+
+        lam = jnp.linalg.eigvalsh(W_scaled)
+
+        n = D.shape[-1]
+
+        logtau = n * jnp.log(self.tau).sum()
+        logdet = jnp.log(1 - self.alpha * lam).sum()
```

Comment:
I think you need `jnp.log1p(-jnp.expand_dims(self.alpha, -1) * lam).sum(-1)` here.